src/nmea.cpp

```diff
@@ -911,6 +914,15 @@ int GPSDriverNMEA::receive(unsigned timeout)
 						   (double)_unicore_parser.heading().heading_deg,
 						   (double)_unicore_parser.heading().heading_stddev_deg,
 						   (double)_unicore_parser.heading().baseline_m);
+
+				} else if (result == UnicoreParser::Result::GotAgrica) {
+					++handled;
```

Comment:
Could be worth adding your comment here to explain the logic:
`This allows to determine whether we are talking to a UM982 and hence request the heading (UNIHEADINGA) message that we actually require.`